app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -162,6 +175,26 @@ public boolean onPreferenceClick(Preference preference) {
         }
     }
 
+    /**
+     * The new Photo Picker with GET_CONTENT takeover redacts location tags
```

Comment:
I would write `On some devices, the new Photo Picker with GET_CONTENT takeover redacts location tags from EXIF metadata.`

Reply:
I've modified it.